frontend/src/component/demo/demo-setup.ts

```diff
@@ -67,75 +68,48 @@ export const gradualRollout = async () => {
+    fetch(
+        formatApiPath(
+            `api/admin/projects/${PROJECT}/features/${featureId}/environments/${ENVIRONMENT}/strategies/${strategyId}`
+        ),
+        { method: 'DELETE' }
+    );
 
-    const { variants }: IFeatureToggle = await fetch(
+const deleteOldStrategies = async (featureId: string) => {
+    const results = await fetch(
         formatApiPath(
-            `api/admin/projects/${PROJECT}/features/${featureId}?variantEnvironments=true`
+            `api/admin/projects/${PROJECT}/features/${featureId}/environments/${ENVIRONMENT}/strategies`
         )
     ).then(res => res.json());
 
-    if (!variants.length) {
-        await fetch(
-            formatApiPath(
-                `api/admin/projects/${PROJECT}/features/${featureId}/environments/${ENVIRONMENT}/variants`
-            ),
-            {
-                method: 'PATCH',
-                headers: {
-                    'Content-Type': 'application/json',
-                },
-                body: JSON.stringify([
-                    {
-                        op: 'add',
-                        path: '/0',
-                        value: {
-                            name: 'red',
-                            weightType: 'variable',
-                            weight: 333,
-                            overrides: [],
-                            stickiness: 'default',
-                            payload: {
-                                type: 'string',
-                                value: 'red',
-                            },
-                        },
-                    },
-                    {
-                        op: 'add',
-                        path: '/1',
-                        value: {
-                            name: 'green',
-                            weightType: 'variable',
-                            weight: 333,
-                            overrides: [],
-                            stickiness: 'default',
-                            payload: {
-                                type: 'string',
-                                value: 'green',
-                            },
-                        },
-                    },
-                    {
-                        op: 'add',
-                        path: '/2',
-                        value: {
-                            name: 'blue',
-                            weightType: 'variable',
-                            weight: 333,
-                            overrides: [],
-                            stickiness: 'default',
-                            payload: {
-                                type: 'string',
-                                value: 'blue',
-                            },
-                        },
-                    },
-                ]),
-            }
+    const tooGenericStrategies = results.filter(
+        (strategy: { constraints: Array<unknown>; segments: Array<unknown> }) =>
```

Comment:
Don't we have the proper types for this instead of `unknown`?

Reply:
we don't care about those types so I left it as unknown. We care about the array lenght